delta/plugins/elasticsearch/src/main/resources/defaults/default-mapping.json

```diff
@@ -182,6 +182,10 @@
         }
       }
     },
+    "keywords": {
+      "type": "object",
```

Comment:
Did you try with the flattened type ?
We have to be careful with the dynamic mapping:
https://www.elastic.co/guide/en/elasticsearch/reference/current/dynamic-field-mapping.html
We don't want someone to push a value that can be parsed as a date/a number and have this weird case where the first occurence set it as such and have regular strings failing after
One alternative to flatten can be mapping templates and:
https://www.elastic.co/guide/en/elasticsearch/reference/current/dynamic-field-mapping.html